ui/src/app/views/workspace-share/component.ts

```diff
@@ -175,9 +175,11 @@ export class WorkspaceShareComponent implements OnInit {
       this.autocompleteNoResults = true;
       return;
     }
-    this.userService.user(this.searchTerm).subscribe((response) => {
-      this.autocompleteLoading = false;
+    const searchTerm = this.searchTerm;
```

Comment:
This variable gets stored via a closure in the below function, copied-by-value so it retains whatever the query was at request time.